src/test/java/org/candlepin/model/CertificateTest.java

```diff
@@ -19,16 +19,24 @@
 
 import org.candlepin.test.DatabaseTestFixture;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import java.util.Date;
 import java.util.List;
 
-import javax.inject.Inject;
 
 public class CertificateTest extends DatabaseTestFixture {
-    @Inject private SubscriptionsCertificateCurator certificateCurator;
-    @Inject private CertificateSerialCurator certSerialCurator;
+    private SubscriptionsCertificateCurator certificateCurator;
+    private CertificateSerialCurator certSerialCurator;
+
+    @BeforeEach
```

Comment:
Same question here, and in several other files

Reply:
I have put all getInstance for curators in the DatabaseTestFixture (plus some other common ones). The get for the Resources and lower use stuff is per test.